src/components/Article/ArticleController.tsx

```diff
@@ -100,6 +90,19 @@ const ArticleController = (props: Props) => {
     }
   }, [editDialogOpen]);
 
+  useEffect(() => {
```

Comment:
below useEffect takes care of all scenarios. This is not necessary and in face might work to reset the search results

Reply:
removed the useEffect